src/units/KM_UnitGroups.pas

```diff
@@ -1232,19 +1231,37 @@ procedure TKMUnitGroup.OrderNone;
 end;
 
 
-//Repeat last order e.g. if new members have joined
-procedure TKMUnitGroup.OrderRepeat;
+//Repeat order after specified aGroup or repeat self order if no aGroup is specified
+procedure TKMUnitGroup.OrderRepeat(aGroup: TKMUnitGroup);
```

Comment:
Lets leave the old OrderRepeat as is and add a new procedure, CopyOrderFrom(aGroup). Less confusing that way and it's a short procedure so not much duplication.